content/docs/user-guide/gems/reference/aws/aws-core/configuring-credentials.md

```diff
@@ -98,8 +126,77 @@ aws configure list
 aws configure list-profiles
 ```
 
+## Controlling Permissions for Users
+All AWS IAM users are given permissions through the attachment of managed or inline IAM polices. See [Changing permissions for an IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_change-permissions.html).
+
+Its recommended that you:
+* Use an Admin user, that is a user with full permissions on an AWS account, when absolutely necessary, such as deploying and provisioning AWS resources.
+* Grant [least privileges](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege) on your AWS users, that is they only have the minimum set of permissions on specific resources they require. This is controlled through setting of managed IAM policies and other mechanisms.
+* As you add and review AWS resources, you update managed policies to grant or revoke the appropriate permissions.
+
+### Create an IAM User Groups
+To make managing permissions easier, its recommended that you create [user groups](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups.html). User groups let you specify permissions for multiple users at time, which can make it easier to manage the permissions for those users.
+
+You can create user groups via the console, the cli or through the CloudFormation/CDK. We recommend you create at least two user groups to work with AWS in O3DE:
+* Admins - Administrator users required to own and manage AWS resources. Typically, performing updates and management of key resources.
+* Users - set of users that take action on the resources as part of normally gameplay or simulation.
+
+#### Create a user group via the AWS Console
+1. Sign in to the AWS Management Console, if you are not already signed in, at https://console.aws.amazon.com/iamv2
```

Comment:
May worth linking to the IAM document for the steps: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups_create.html

Reply:
Sure, I link to this elsewhere, but can add it here.